test/quic-openssl-docker/run_endpoint.sh

```diff
@@ -69,8 +69,14 @@ if [ "$ROLE" == "client" ]; then
        exit 0
        ;;
     "chacha20")
-       OUTFILE=$(basename $REQUESTS)
-       SSL_CERT_FILE=/certs/ca.pem curl --verbose --tlsv1.3 --tls13-ciphers TLS_CHACHA20_POLY1305_SHA256 --http3 -o /downloads/$OUTFILE $REQUESTS || exit 1
+       for req in $REQUESTS
+       do
+           OUTFILE=$(basename $req)
+           echo -n "$OUTFILE " > ./reqfile.txt
```

Comment:
Was this supposed to *accumulate* all the output file names, or capture just the last one? If not accumulate, why the trailing <SPACE>?
```suggestion
           printf "%s " "$OUTFILE" >> ./reqfile.txt
```